apps/web/utils/actions.ts

```diff
@@ -109,7 +125,10 @@ export async function deleteAccountAction() {
     captureException(error);
   }
 
-  await prisma.user.delete({ where: { email: session.user.email } });
+  if (lemonSqueezySubscriptionId)
+    await cancelSubScriptionForUser(lemonSqueezySubscriptionId);
+
+  // await prisma.user.delete({ where: { email: session.user.email } });
```

Comment:
Uncomment. We do want to delete the user.

(We do want to keep the user in Lemon though)